git-blame-ignore-revs

```diff
@@ -6,3 +6,5 @@
 4f396cceb84d79d5ef540e91c1a8420e8de74a56
 # reformat lib-ee/emqx_enterprise_conf
 1aa82992616ad848539a533a5cd20ba6f9071e5a
+# reformat apps/emqx_gateway
+766a121a7ad9d1a600cbafb3f4a8ee8d5d7369d5
```

Comment:
This commit id doesn't exists.
The format PR should be splitd with at least two commits.
And make sure the `git-blame-ignore-revs` refs to the correct commit.